tests/testthat/test_kernel.r

```diff
@@ -0,0 +1,6 @@
+context('kernel')
+
+status <- system2('python3', 'test_ir.py', wait=TRUE)
```

Comment:
I guess we'll just remove this file from the tarball we submit to CRAN, in that case :-(